src/utilities/rois.ts

```diff
@@ -16,18 +16,23 @@ export function createCommitedRoi<T>(
   id: string,
   options: Omit<Partial<CommittedRoi<T>>, 'id'> = {},
 ): CommittedRoi<T> {
+  const hasClassNames =
+    options.className !== undefined || options.selectedClassname !== undefined;
+
   return {
     id,
     label: '',
     ...createInitialBox(),
-    style: {
-      backgroundColor: 'black',
-      opacity: 0.5,
-    },
-    selectedStyle: {
-      backgroundColor: 'blue',
-      opacity: 0.5,
-    },
+    ...(!hasClassNames && {
```

Comment:
Does this means if I pass a `selectedClassname` but no `className`, my default styles don't get applied?

Shouldn't you make this check separately for normal vs selected styles?

Reply:
On the main branch it looks like, if we give {} to style or selectedStyle, the Box is rendered with nothing. Right @josoriom ? 

If yes, we have some solution about the API, what do you want to accept as props for style/selectedStyle and className/selectedClassName

1. Both required like, className **AND** selectedClassName **OR** style **AND** selectedStyle
2. Can be Shuffled like, className **AND** selectedStyle **OR** style **AND** selectedClassName
3. Can be linked with an undefined value like, style and undefined (for selectedClassName or selectedStyle) **OR** undefined and a value.

I think, proposition 1 looks better, but we have to everytime, give both value (for selected and non-selected style/className)

Reply:
@Sebastien-Ahkrin  Yes, that's because it overwrites the default values. 

Giving priority to className as you did is fine since in the current status if you define className, it will be overridden by the default style. However, as suggested by @stropitek, handling it separately offers flexibility. (i.e., when selectedClassName exists, the default selectedStyle should not be applied). In such case, if you define className without selectedClassName, the default selectedStyle will be applied and in the other way.  Therefore, you can use any combination.